### ISSUE TITLE
/characters/{character_id}/wallet/journal/ does not contain the balance when journal is empty

### ISSUE THREAD

**Issue by wvdvegt** [esi/esi-issues#1259]
# Bug

Because the balance field is missing when the journal is empty it is impossible to retrieve the balance in the dev branch. (latest 'still' has another endpoint that returns the personal wallet balance.

It happens when a character has not made any transactions in the last 30 days (according to esi docs).

It is unclear which version to use, latest or dev (which is the latest one?).

## Request

`GET /characters/{character_id}/wallet/journal/

## Response

Empty Array (so a workaround for previous versions to take the balance field from the first array item now fails.

### Status Code

`200`

## Expected

At least the current balance to be present or a separate end-point as /characters/{character_id}/wallet/

# Checklist

Check all boxes that apply to this issue:

- [x] Bug description is provided
- [x] Request path is provided
- [x] Response status code is provided
- [ ] Response headers are provided
- [ ] Response body is provided
- [x] Expected response is provided

**Comment by Blacksmoke16** [comment]
The journal endpoint isn't intended to be used to get your current balance.  You should be using https://esi.evetech.net/ui/?version=latest#/Wallet/get_characters_character_id_wallet.

> It is unclear which version to use, latest or dev (which is the latest one?).

You should be using neither of those.  See https://developers.eveonline.com/blog/article/esi-endpoint-versioning-important-info-and-best-practices.

tl;dr you should be using a specific version, not `latest`, or `dev` alises.  I.e. `v1`.
